jenkins/docker/python.Dockerfile

```diff
@@ -18,7 +18,13 @@ RUN pip install poetry
 # sdk dependencies
 RUN apt-get install -y zbar-tools
 
+# codecov uploader
+RUN curl -Os https://uploader.codecov.io/latest/linux/codecov \
+	&& chmod +x codecov \
+	&& mv codecov /usr/local/bin
+
 # add ubuntu user (used by jenkins)
 RUN useradd --uid 1000 -ms /bin/bash ubuntu
+ENV PATH=$PATH:/home/ubuntu/.local/bin
```

Comment:
why is this needed in py docker but not js docker? 🤔

Reply:
I saw some warnings when installing the python packages

Reply:
@gimre-xymcity - any idea?

Reply:
hmm not really, do you perhaps have error saved?

Reply:
@gimre-xymcity This was the warning I saw in the logs

```
13:47:33  + python3 -m pip install -r /home/ubuntu/jenkins/workspace/l_generated_monorepo_parser_main/linters/python/lint_requirements.txt
13:47:35    WARNING: The scripts isort and isort-identify-imports are installed in '/home/ubuntu/.local/bin' which is not on PATH.
13:47:35    Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
```